    MFC r368588:
    
    pf: Allow net.pf.request_maxcount to be set from loader.conf
    
    Mark request_maxcount as RWTUN so we can set it both at runtime and from
    loader.conf. This avoids users getting caught out by the change from tunable to
    run time configuration.
    
    Suggested by:   Franco Fichtner
    
    (cherry picked from commit 08d13750ebdae45bcdb73d52665b823e9ba93db1)
--- sys/netpfil/pf/pf.c.orig
+++ sys/netpfil/pf/pf.c
@@ -382,7 +382,7 @@
     &pf_hashsize, 0, "Size of pf(4) states hashtable");
 SYSCTL_ULONG(_net_pf, OID_AUTO, source_nodes_hashsize, CTLFLAG_RDTUN,
     &pf_srchashsize, 0, "Size of pf(4) source nodes hashtable");
-SYSCTL_ULONG(_net_pf, OID_AUTO, request_maxcount, CTLFLAG_RW,
+SYSCTL_ULONG(_net_pf, OID_AUTO, request_maxcount, CTLFLAG_RWTUN,
     &pf_ioctl_maxcount, 0, "Maximum number of tables, addresses, ... in a single ioctl() call");
 
 VNET_DEFINE(void *, pf_swi_cookie);
